src/sentry/options/defaults.py

```diff
@@ -1846,6 +1846,13 @@
     default=[],
     flags=FLAG_ALLOW_EMPTY | FLAG_AUTOMATOR_MODIFIABLE,
 )
+# Used for the z-score when calculating the margin of error in performance
+register(
+    "performance.confidence.z-score",
```

Comment:
The name of this option should probably contain the word extrapolation - so we can make sense of where this comes from.